ph5/utilities/ph5validate.py

```diff
@@ -847,14 +831,33 @@ def get_args():
 
     args = parser.parse_args()
 
-    # Add formatter
-    args.level = str(args.level).upper()
-    if args.level != "ERROR" and args.level != "WARNING" and \
-            args.level != "INFO":
-        raise ValueError("Invalid logging level.")
+    outfile = args.outfile
```

Comment:
```suggestion

```

Reply:
You was correct about args.outfile already has the default 'ph5_validate.log so I change line 834. But I don't understand why you want to remove the line 834 which defines  variable outfile. When I removed that as you suggested, I got the error "global name 'outfile' is not defined".